core/tooltip.js

```diff
@@ -111,6 +111,13 @@ Blockly.Tooltip.MARGINS = 5;
  */
 Blockly.Tooltip.DIV = null;
 
+/**
+ * Container element to render the tooltip into.
+ * @type {?Element}
+ * @const
```

Comment:
These aren't const if we expect them to get set externally.

Reply:
Removed.